assets/src/edit-story/components/form/dropDown/list.js

```diff
@@ -0,0 +1,258 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useState, useEffect, useCallback, useMemo, useRef } from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { rgba } from 'polished';
+import { useDebouncedCallback } from 'use-debounce';
+
+/**
+ * Internal dependencies
+ */
+import { useKeyDownEffect } from '../../keyboard';
+import useFocusOut from '../../../utils/useFocusOut';
+
+const ListContainer = styled.div`
+  float: right;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  width: 100%;
+  min-width: 160px;
+  max-height: 370px;
+  overscroll-behavior: none auto;
+`;
+
+const List = styled.ul.attrs({ role: 'listbox' })`
+  width: 100%;
+  padding: 5px 0;
+  margin: 2px 0 0;
+  font-size: 14px;
+  text-align: left;
+  list-style: none;
+  background-color: ${({ theme }) => theme.colors.fg.v1};
+  background-clip: padding-box;
+  border-radius: 4px;
+  box-shadow: 0 6px 12px ${({ theme }) => rgba(theme.colors.bg.v0, 0.175)};
+`;
+
+const Item = styled.li.attrs({ tabIndex: '0', role: 'option' })`
+  letter-spacing: ${({ theme }) => theme.fonts.label.letterSpacing};
+  padding: 16px;
+  margin: 0;
+  font-family: ${({ theme }) => theme.fonts.label.family};
+  font-size: ${({ theme }) => theme.fonts.label.size};
+  line-height: ${({ theme }) => theme.fonts.label.lineHeight};
+  font-weight: ${({ theme }) => theme.fonts.label.weight};
+
+  &:hover {
+    background-color: ${({ theme }) => theme.colors.bg.v12};
+  }
+
+  &:focus {
+    background-color: ${({ theme }) => theme.colors.bg.v12};
+    outline: none;
+  }
+`;
+
+const availableKeysForSearch = [
```

Comment:
Nit pic. Wouldn't be clearer to do something like, `const availableKeysForSearch = "abcdefghijklmnopqrstuvwxyz123456789".split()`

Reply:
Good one! Thanks, fixed from the last commit.